cli/cmd/component.go

```diff
@@ -404,3 +419,74 @@ func runComponentsDelete(_ *cobra.Command, args []string) (err error) {
 	cli.OutputHuman("Reach out to us at %s\n", color.HiCyanString("support@lacework.net"))
 	return
 }
+
+func runComponentsDevMode(_ *cobra.Command, args []string) error {
+	cli.StartProgress("Loading components state...")
+	var err error
+	cli.LwComponents, err = lwcomponent.LoadState(cli.LwApi)
+	cli.StopProgress()
+	if err != nil {
+		return errors.Wrap(err, "unable to load components")
+	}
+
+	component, found := cli.LwComponents.GetComponent(args[0])
+	if !found {
+		component = &lwcomponent.Component{
+			Name: args[0],
```

Comment:
Should there be any restrictions on a component name. For example I could pass in "!"